lib/src/base-styles/_nx-btn.scss

```diff
@@ -7,111 +7,122 @@
 @import '../scss-shared/nx-variables';
 @import '../scss-shared/nx-text-helpers';
 
+%nx-btn--disabled {
+  &, &:hover, &:focus {
+    background-color: #e9e9e9;
+    border-color: #949494;
+    color: #949494;
+    cursor: default;
+  }
+}
+
 .nx-btn {
   @include semi-bold();
   @include container-horizontal;
 
-  background-color: $nx-grey-1;
-  border: $nx-border-dark;
-  border-radius: 4px;
-  color: $nx-text-color-light;
+  background-color: $nx-white;
+  border: 1px solid #045dd1;
+  border-radius: 6px;
+  color: #045dd1;
   cursor: pointer;
   display: inline-block;
-  font-size: 14px;
-  height: 32px;
+  font-size: 16px;
   line-height: 16px;
```

Comment:
This should probably be gotten rid of and the vertical padding decreased to 8px

Reply:
> This should probably be gotten rid of and the vertical padding decreased to 8px

Interesting side-effect, button is now 40px tall: https://github.com/sonatype/sonatype-react-shared-components/pull/39/commits/fff7ae93ed8ea9dfefb1e1b544413219a37e8032